Update battery documentation (#319)
icub-tech-iit/documentation: docs/battery_boards/bat_board/bat_protocol.md

- DCDC_status_B:

    Position | BIT[3] | BIT[2] | BIT[1] | BIT[0] |
    |:---|:---:|:---:|:---:|:---:|
    Value | HSM_SW_F | HSM_HW_F | PB1_restart | PB2_restart |

>     Description | OC Fault on the HSM triggered by overcoming the MAX current defined in FW | OC Fault on the HSM triggered by FLT Pin on the HSM micro | Restart phase of the push button 1 | Restart phase of the push button 2 |

Reviewer: ```suggestion
    Description | OC Fault on the HSM triggered by overcurrent (threshold defined in the FW) | OC Fault on the HSM triggered by FLT Pin on the HSM micro | Restart phase of the push button 1 | Restart phase of the push button 2 |
```

    Possible status | FAULT_OFF(0)/FAULT_ON(1) | FAULT_OFF(0)/FAULT_ON(1) | Start-up phase(1)/stable operation(0) | Start-up phase/stable operation |

- Final status shown at the port is equal to: 

    `(DCDC_status_B << 8 ) | DCDC_status_A`

> Therefore the end user will see a decimal number on the Display of the BAT, that can be translated in BITs and analyzed as defined by the tables above and summmed up here:

Reviewer: ```suggestion
Thereby, the end user sees a decimal number on the BAT Display, which can be transformed in BITs and analyzed as described below:
```
